src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php

```diff
@@ -20,22 +20,28 @@
 
 namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Dto;
 
+use function array_key_exists;
+
 class AttributeForm
 {
     public $languages = [];
 
-    private function __construct(
-        array $languages
-    ) {
+    public $excludeFrom = [];
+
+    private function __construct(array $languages, array $excludeFrom)
+    {
         foreach ($languages as $language => $values) {
             $this->languages[$language] = AttributeTypeInformation::fromLanguage($values, $language);
         }
+        $this->excludeFrom = $excludeFrom;
     }
 
-    public static function fromForm(array $languages): ?AttributeForm
+    public static function fromForm(array $form): ?AttributeForm
     {
-        return new self(
-            $languages
-        );
+        $excludeFrom = [];
+        if (array_key_exists('excludeOnEntityType', $form)) {
```

Comment:
Is it justified to use a constant for 'excludeOnEntityType'?

Reply:
Good suggestion. At this point, this is the only place where we consume that key in the application. And it should remain the only place. I do not see the need to use a class constant in this case.